awx_collection/plugins/modules/tower_workflow_launch.py

```diff
@@ -20,11 +20,34 @@
     - Launch an Ansible Tower workflows. See
       U(https://www.ansible.com/tower) for an overview.
 options:
-    workflow_template:
+    name:
       description:
         - The name of the workflow template to run.
       required: True
       type: str
+      aliases:
+        - workflow_template
```

Comment:
👍 this looks smart to me. One question that still nags me a little bit is if you can provide an id for this. Even if that is allowed, it would be an abuse of parameters. If it is supported, I think a test would be good.